dev/local/Makefile

```diff
@@ -101,6 +101,37 @@ web:
 		--network delphi-net --name delphi_web_epidata \
 		delphi_web_epidata >$(LOG_WEB) 2>&1 &
 
+.PHONY=web_debug
+web_debug:
+	@# Stop container if running
+	@if [ $(WEB_CONTAINER_ID) ]; then\
+		docker stop $(WEB_CONTAINER_ID);\
+	fi
+
+	@# Setup virtual network if it doesn't exist
+	@docker network ls | grep delphi-net || docker network create --driver bridge delphi-net
+
+	@# Build the web_epidata image
+	@cd repos/delphi/delphi-epidata; \
+		docker build -t delphi_web_epidata \
+			$(M1) \
+			-f ./devops/Dockerfile .;\
+		cd -
+
+	@# Run the web server
+	@# MODULE_NAME specifies the location of the `app` variable, the actual WSGI application object to run.
+	@# see https://github.com/tiangolo/meinheld-gunicorn-docker#module_name
+	@docker run --rm -p 127.0.0.1:5000:5000 -p 127.0.0.1:5678:5678 \
+		$(M1) \
+		--mount type=bind,source=$(CWD)repos/delphi/delphi-epidata/devops/start.sh,target=/start.sh,readonly \
```

Comment:
```suggestion
                `# mount the debuging start script into the container so the webserver uses it ` \
		--mount type=bind,source=$(CWD)repos/delphi/delphi-epidata/devops/start_vscodedebug.sh,target=/start.sh,readonly \
```
to support renaming the local `start.sh` file and make a note so its clearer as to whats happening

Reply:
actually, im not sure that bash "trick" i used to add an in-line comment will work inside the makefile...